.github/workflows/workflow_tests.yml

```diff
@@ -23,13 +44,79 @@ jobs:
           architecture: 'x64'
```

Comment:
could we add test matrix to python-version that 3.10, 3.11, 3.12 will all be tested?

Reply:
It has been tested that some libraries in Python 3.11 have changed and are not supported